### PR TITLE
Add msgpack incompatible warning to 3000 release notes

### PR DESCRIPTION
**Pull request by Ch3LL**

### CODE REVIEW

**Review by waynew — APPROVED** [review]

File: doc/topics/releases/3000.rst [hunk 1]
```diff
@@ -33,6 +33,12 @@ we continue to work on upgrading Tornado to a recent version. New code that
 uses Tornado should import the module from ``salt.ext.tornado`` instead of
 importing the system version of Tornado.
 
+Msgpack 1.0.0rc1 Incompatibility
+================================
+Salt is currently incompatible with mgspack 1.0.0rc1. We recommend using
+versions <= 1.0.0rc1. This impacts the Salt Master's ability to run commands
```
Comment (s0undt3ch) [inline doc/topics/releases/3000.rst]: versions < 1.0

**Review by s0undt3ch — CHANGES_REQUESTED** [review]

File: doc/topics/releases/3000.rst (hunk 1 above)
Comment (dwoz) [reply doc/topics/releases/3000.rst]: > versions < 1.0

this

Comment (Ch3LL) [reply doc/topics/releases/3000.rst]: thanks for catching that. updated. i put 1.0.0 instead of 1.0 as that looks to be the versioning scheme

Comment (s0undt3ch) [reply doc/topics/releases/3000.rst]: :+1:

**Review by s0undt3ch — APPROVED** [review]

**Review by garethgreenaway — APPROVED** [review]

**Review by sagetherage — APPROVED** [review]

**Review by twangboy — APPROVED** [review]

**Pull request merged**